Specs/NumberWithUnit/French/CurrencyModel.json

```diff
@@ -1533,5 +1533,37 @@
     "Input": "pour cent",
     "NotSupported": "javascript,python,java",
     "Results": []
+  },
+  {
+    "Input": "Je veux un crédit de EUR 1000 sur 3 ans",
```

Comment:
Can you add both the English and French specs to DateTimeModel and ComplexCalendar too, to make sure the year durations are correctly extracted there?